scripts/build_gnome_supp.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ ! -d ${DIR}/tmp/GNOME.supp ]; then
+    mkdir -p ${DIR}/tmp;
```

Comment:
No ; needed.

Reply:
Actually, not worth change itself. But, there are missing `"...${DIR}..."`.